.ci/riscv-toolchain-install.sh

```diff
@@ -8,10 +8,15 @@ check_platform
 
 mkdir -p toolchain
 
-# GNU Toolchain for RISC-V
-GCC_VER=2024.04.12
-TOOLCHAIN_REPO=https://github.com/riscv-collab/riscv-gnu-toolchain/releases
+if [[ "$#" = 0 ]] || [[ "$1" != "riscv-collab" ]]; then
```

Comment:
`"$#" = 0` is missing one '=' ?

Reply:
No, but the quotation marks are missing here. In bash, both single and double equals sign (`=`) are used to string comparison, or use `-eq` for simplicity.

Reply:
Consider `"$#" = "0"` or `$# == 0`. I tried both, both have the same effect. 

> No, but the quotation marks are missing here. In bash, both single and double equals sign (`=`) are used to string comparison, or use `-eq` for simplicity.

Reply:
After testing, the original bash script seems to work too.